samples/nrf9160/modem_shell/src/main.c

```diff
@@ -221,6 +223,20 @@ void main(void)
 
 	mosh_print_version_info();
 
+#if defined(CONFIG_NRF_CLOUD_REST) || defined(CONFIG_NRF_CLOUD_MQTT)
+	/* Due to iperf3, we cannot let nrf cloud lib to initialize cJSON lib to be
+	 * using kernel heap allocations (i.e. k_ prepending functions).
+	 * Thus, we use standard c alloc/free, i.e. "system heap".
+	 */
+	struct nrf_cloud_os_mem_hooks hooks = {
+		.malloc_fn = malloc,
+		.calloc_fn = calloc,
+		.free_fn = free,
+	};
+
+	nrf_cloud_os_mem_hooks_init(&hooks);
```

Comment:
It would be nice to make selection between k_malloc and malloc with a Kconfig option. Was there an obvious reason not to do it with Kconfig in nrf_cloud library?

Reply:
That would have been one option, but was now done in this way. This makes also possible to have own implementation of those.